library/src/main/java/com/smartnsoft/webviewfragment/WebViewFragment.java

```diff
@@ -343,6 +305,40 @@ public void onClick(View view)
     }
   }
 
+  protected void onRetrieveBusinessObjects()
```

Comment:
@ludovicroland Why did you keep these two legacy method names ? It would be rather confusing for someone using this class, don't you think ?

Reply:
@n0tnull : It allows developers to override the default behavior of theses methods if needed with no need to override the `onCreate` method. Feel free to rename these method if you want.